package/MDAnalysis/coordinates/chain.py

```diff
@@ -211,7 +212,19 @@ def convert_pos_to_native(self, x):
         return self.active_reader.convert_pos_to_native(x)
 
     def copy(self):
-        raise NotImplementedError("Copy not implemented for ChainReader")
+        #raise NotImplementedError("Copy not implemented for ChainReader")
+        new = self.__class__(self.filenames)
+        new.readers=[reader.copy() for reader in self.readers]
+        new.n_atoms = self._get_same('n_atoms').copy()
```

Comment:
I don't think setting `n_atoms` `n_frames` etc is necessary.  The new Reader can work these out the same way as this Reader does